### ISSUE TITLE
Errata Page 522 ch15

### ISSUE THREAD

**Issue by jadeiX** [rasbt/python-machine-learning-book-3rd-edition#149]
@rasbt 
The following equation on page 522:
![image](https://user-images.githubusercontent.com/32917209/107130787-ea8d4d00-6885-11eb-86cf-d2f159ee7753.png)
Assuming array index of _**x and w**_ starts at 0 (which is suggested by the fact that the index k starts at 0) the equation will give for y[0] and m=4 and p=0:
`y[0]=x[4]w[0] + x[3]w[1] + x[2]w[2] + x[1]w[3]`
thereby skipping `x[0]`
i would suggest the following correction
![image](https://user-images.githubusercontent.com/32917209/107130891-cf6f0d00-6886-11eb-9570-2eff3b877bbb.png)
Likewise
![image](https://user-images.githubusercontent.com/32917209/107130903-f3cae980-6886-11eb-837a-10e60baa0b1b.png)
And the example should be corrected to
![image](https://user-images.githubusercontent.com/32917209/107130921-13621200-6887-11eb-87e3-2658b53adbee.png)

**Comment by rasbt** [comment]
Good catch. Sorry about that and thanks for the feedback. We will fix that in the next edition
